Xamarin.Forms.Platform.Android/Renderers/SwipeViewRenderer.cs

```diff
@@ -228,9 +233,29 @@ public override bool OnTouchEvent(MotionEvent e)
 			return true;
 		}
 
-		public override bool OnInterceptTouchEvent(MotionEvent ev)
+		public override bool OnInterceptTouchEvent(MotionEvent e)
 		{
-			return true;
+			switch (e.Action)
+			{
+				case MotionEventActions.Down:
+					_downX = e.RawX;
+					_downY = e.RawY;
+					_initialPoint = new APointF(e.GetX() / _density, e.GetY() / _density);
+					break;
+				case MotionEventActions.Move:
+					float x = Math.Abs((_downX - e.GetX()) / _density);
+					float y = Math.Abs((_downY - e.GetY()) / _density);
+
+					if (x > 10f || y > 10f)
```

Comment:
Please move 10f to constants

Is it worth to create a dependency property so the user can set the minimum delta to process swiping interactions?

Reply:
The value is used in several places, I have created a constant.
About creating a dependency property, there are several ideas for properties that we could add in future versions (`SwipeThreshold`, `SwipeMinimumDelta`, etc.). Remember that you can create a Spec with your proposal!.